src/main/resources/application.yml

```diff
@@ -22,6 +22,8 @@ spring:
       acknowledge-mode: CLIENT
 
 labresults:
+  trustStoreUrl: ${LAB_RESULTS_SSL_TRUST_STORE_URL:}
+  trustStorePassword: ${LAB_RESULTS_SSL_TRUST_STORE_PASSWORD:}
```

Comment:
Trailing colon can be removed

```
  trustStoreUrl: ${LAB_RESULTS_SSL_TRUST_STORE_URL}
  trustStorePassword: ${LAB_RESULTS_SSL_TRUST_STORE_PASSWORD}
```

Reply:
Trailing colon means that it's optional and empty value will be used if nothing is provided
Do you think this will work without colon as well?

Reply:
I think so. The colon is redundant here